XCUITest/CopyTest.swift

```diff
@@ -7,13 +7,14 @@ import XCTest
 // Note: this test is tested as part of the base test case, and thus is disabled here.
 
 class CopyTest: BaseTestCase {
-    func testCopyMenuItem() {
+    func testCopyMenuItem() throws {
+        throw XCTSkip("This test needs to be updated or removed")
```

Comment:
Can't find the way to show the copy option from url bar, we had similar issue on firefox-ios. I would say to disable for now while we find a good an stable way to do this.